homeassistant/components/heatzy/translations/fr.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config": {
+        "flow_title": "{name}",
+        "step": {
+            "user": {
+                "title": "Enregistrer votre compte Heatzy.",
+                "data": {
+                    "username": "Email",
+                    "password": "Mot de passe"
+                }
+            }
+        },
+        "error": {
+            "cannot_connect": "Une erreur inconnue s'est produite."
+        },
+        "abort": {
+            "already_configured": "Votre compte est déjà enregistré."
+        }
+    }
+}
```

Comment:
I believe you can remove this file, as all other translations are automatically generated by Home Assistant.